src/VisualStudio/CSharp/Impl/Options/AutomationObject.cs

```diff
@@ -515,16 +515,16 @@ public int Space_WithinSquares
             set { SetBooleanOption(CSharpFormattingOptions.SpaceWithinSquareBrackets, value); }
         }
 
-        public int Style_PreferIntrinsicPredefinedTypeKeywordInDeclaration
+        public string Style_PreferIntrinsicPredefinedTypeKeywordInDeclaration
```

Comment:
This is hte automation object right?  I think it can be accessed through the DTE.